news-homepage-main/style.css

```diff
@@ -0,0 +1,116 @@
+
+*{
+    font-family: 'Inter', sans-serif;
+    box-sizing: border-box;
+}
+
+body{
+    padding-left: 50px;
+}   
+
+
+
+
+.nav-bar{
+    display: flex;
+    justify-content: space-between; 
+    margin-bottom: 50px;
+}
+
+.nav-container{
+    display: flex;
+    gap: 30px;
+}
+
+.main-photo{
+    background-image: url(./assets/images/image-web-3-desktop.jpg);
+    height: 300px;
+    width: 700px;
+    background-size: cover;
+    background-position: center;
+}
+
+.container{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 20px;   
+}
+
+.section1{
+    width: 350px;
+    background-color:hsl(240, 100%, 5%) ;
+    padding: 20px; 
+    height: 470px;   
+}
+
+.section-title{
+    color:hsl(35, 77%, 62%) ;
+}
+
+.child{
+    margin-bottom: 50px;
+    border-bottom:  1px solid white;
+}
+.child1{
+    display: flex;
+    gap: 20px;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
+.child2{
+    width: 340px;
+}
+.child-title{
+    font-size: 40px;
+    margin: 0;
+    color:hsl(240, 100%, 5%) ; 
+    width: 250px;
+    padding-right: 30px;
+}
+.child-button{
+    width: 120px;
+    height: 30px;
+    background-color: hsl(5, 85%, 63%);
+    border: none;
+    color: white;
+}
+
+h3{
+    color:hsl(36, 100%, 99%) ;
+    margin: 0;
+    line-height: 0;
+}
+
+.text{
+    color: hsl(236, 13%, 42%) ;
+    margin-bottom: 40px;
+}
+.text1{
+    color: hsl(236, 13%, 42%) ; 
+}
+
+.section2{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    
+    
+}
+.image{
+    width: 120px;
```

Comment:
Nu poti sa ii dai si width si height, o sa distorsionezi imaginea. Ii dai la container un width si height daca vrei si la imagine width 100%, sau doar ii dai un width la imagine ca height-ul se regleaza automat. Sau pui o imagine in background.